.github/ISSUE_TEMPLATE/Bug_Report.yml

```diff
@@ -23,7 +23,7 @@ body:
   - type: textarea
     attributes:
       label: Steps to reproduce
-      placeholder: A description of how to trigger this bug
+      placeholder: A description of how to trigger this bug.
```

Comment:
Since "A description of how to trigger this bug" is a fragment and not a complete sentence it didn't include a period.

Either make it a sentence or eliminate the period.

Maybe "Include a description of how to trigger this bug." or "Add instructions to reproduce this bug."

Reply:
In all other placeholders, there are punctuation marks